src/status_im/native_module/impl/module.cljs

```diff
@@ -120,9 +120,22 @@
   (when status
     (call-module #(.discardTransaction status id))))
 
+(defn execute-js [bot-id js-code callback]
```

Comment:
There are a few more places to add this for new bindings, you can see an example here (check src file changes): https://github.com/status-im/status-react/commit/6d5251ad2072c949c0608acde4144d4f4cff0d11

Reply:
thanks a lot! I'll take a look and update my PR!

Reply:
added this binding to the places as in the mentioned commit